onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -1892,9 +1885,9 @@ void test_reduce_apis(size_t size) {
   cudaFree(device_output_square_sum);
   cudaFree(device_output_mean);
 
-  EXPECT_TRUE(std::abs(output_sum - expected_output_sum) / expected_output_sum < 1e-4f);
-  EXPECT_TRUE(std::abs(output_square_sum - expected_output_square_sum) / expected_output_square_sum < 1e-4);
-  EXPECT_TRUE(std::abs(output_mean - expected_output_mean) / expected_output_mean < 1e-4f);
+  EXPECT_LT(std::abs(output_sum - expected_output_sum) / expected_output_sum, tolerance);
```

Comment:
Would "ASSERT_NEAR(val1, val2, abs_error);" be better?

Reply:
this is actually a relative error test instead of absolute error: (a-b)/b < tolerance
i can pick a better name for `tolerance`